lib/undercover/plugin.rb

```diff
@@ -23,15 +23,15 @@ class DangerUndercover < Plugin
     # If there are reports then it shows the report as a warning in danger.
     # @return  [void]
     #
-    def report(undercover_path, sticky: true)
+    def report(undercover_path = 'coverage/undercover.txt', sticky: true)
```

Comment:
One minor improvement for code discovery would be to have a constant for this:

```ruby
DEFAULT_PATH = 'coverage/undercover.txt'.freeze

def report(undercover_path = DEFAULT_PATH, sticky: true)
```

Reply:
Fixed in [8bd4983](https://github.com/nimblehq/danger-undercover/pull/3/commits/8bd498350b37b703057501752ef36f8f6413bf0a)